### PR TITLE
[YUNIKORN-901] Update the website for v0.12

### PR DESCRIPTION
**Pull request by yuchaoran2011**
* Renamed the doc files for 0.12.0 to 0.12.1
* Updated the Roadmap
* Added the release announcement for 0.12.1

### CODE REVIEW

**Comment by yangwwei** [comment]
Hi @yuchaoran2011 , it looks like the docker image link in the download page for 0.12.1 is wrong, still points to 0.11. Could you please fix that?

File: src/pages/community/download.md [hunk 1]
```diff
@@ -29,14 +29,15 @@ The downloads for the latest release are distributed via mirror sites.
...
+| v0.12.1 | 2021-12-17   | [Download](https://www.apache.org/dyn/closer.lua/incubator/yunikorn/0.12.1/apache-yunikorn-0.12.1-incubating-src.tar.gz) <br />[Checksum](https://downloads.apache.org/incubator/yunikorn/0.12.1/apache-yunikorn-0.12.1-incubating-src.tar.gz.sha512) & [Signature](https://downloads.apache.org/incubator/yunikorn/0.12.1/apache-yunikorn-0.12.1-incubating-src.tar.gz.asc)                            | [scheduler](https://hub.docker.com/layers/apache/yunikorn/scheduler-0.11.0/images/sha256-7d156e4df2cb1a99d6f3cf5bfd15ae42c7c195f66411b83a720b375194209d20) <br />[admission-controller](https://hub.docker.com/layers/apache/yunikorn/admission-0.11.0/images/sha256-2b1cee3e79a0f08c835ed264c537b14eb0527d7196dcbbf613296f034c8c2a70 )<br />[web](https://hub.docker.com/layers/apache/yunikorn/web-0.11.0/images/sha256-e07a8465fefb4f51ab989b7be4db824b51fc4b925fb400c09fad87d0b0729246) | [Announcement](/release-announce/0.11.0) |
```
Comment (wilfred-s) [inline src/pages/community/download.md:36]: * all image links point to 0.11.0 should be 0.12.1
* announcement points to 0.11.0
* The date for release should be updated to the date the IPMC approves the release, that is when we officially release the code

File: src/pages/community/download.md [hunk 2]
```diff
@@ -29,14 +29,15 @@ The downloads for the latest release are distributed via mirror sites.
...
+| v0.11.0 | 2021-08-18   | [Download](https://www.apache.org/dyn/closer.lua/incubator/yunikorn/0.11.0/apache-yunikorn-0.11.0-incubating-src.tar.gz) <br />[Checksum](https://downloads.apache.org/incubator/yunikorn/0.11.0/apache-yunikorn-0.11.0-incubating-src.tar.gz.sha512) & [Signature](https://downloads.apache.org/incubator/yunikorn/0.11.0/apache-yunikorn-0.11.0-incubating-src.tar.gz.asc)                            | [scheduler](https://hub.docker.com/layers/apache/yunikorn/scheduler-0.11.0/images/sha256-7d156e4df2cb1a99d6f3cf5bfd15ae42c7c195f66411b83a720b375194209d20) <br />[admission-controller](https://hub.docker.com/layers/apache/yunikorn/admission-0.11.0/images/sha256-2b1cee3e79a0f08c835ed264c537b14eb0527d7196dcbbf613296f034c8c2a70 )<br />[web](https://hub.docker.com/layers/apache/yunikorn/web-0.11.0/images/sha256-e07a8465fefb4f51ab989b7be4db824b51fc4b925fb400c09fad87d0b0729246) | [Announcement](/release-announce/0.11.0) |
```
Comment (wilfred-s) [inline src/pages/community/download.md:37]: The link for the source code for all releases except the latest should point to the archive. That means this link needs to be updated: `https://www.apache.org/dyn/closer.lua/` and should become `https://archive.apache.org/dist/`
Rest stays the same for the 0.11.0 release

File: src/pages/community/roadmap.md [hunk 3]
```diff
@@ -25,32 +25,27 @@ under the License.
...
 
-| Version 	| Release Date  	| Release Manager 	    | Release Note         	                             |
-|---------	|:-------------:	|:---------------:	    |----------------------	                             |
-| 0.11.0  	| 2021-08-18    	| Kinga Marton        	    | [0.11.0-release-notes](../release-announce/0.11.0) |
-| 0.10.0  	| 2021-04-09    	| Tao Yang        	    | [0.10.0-release-notes](../release-announce/0.10.0) |
-| 0.9.0   	| 2020-08-28        | Wilfred Spiegelenburg | [0.9.0-release-notes](../release-announce/0.9.0)   |
-| 0.8.0   	| 2020-05-04	    | Weiwei Yang     	    | [0.8.0-release-notes](../release-announce/0.8.0)   |
+| Version 	 | Release Date  	 |   Release Manager 	   | Release Note         	                             |
+|-----------|:---------------:|:---------------------:|----------------------------------------------------|
+| 0.12.1  	 | 2021-12-17    	 |  Chaoran Yu        	  | [0.12.1-release-notes](../release-announce/0.12.1) |
```
Comment (wilfred-s) [inline src/pages/community/roadmap.md:47]: The date should be set to the IPMC vote result

**Review by wilfred-s — CHANGES_REQUESTED** [review]
minor changes to comply with Apache rules

**Comment by yuchaoran2011** [comment]
@wilfred-s @yangwwei Thanks for catching the errors. Now they are fixed. Please review again

**Review by wilfred-s — APPROVED** [review]
One nit that should be fixed on commit: the release date in the `release-announce/0.12.1.md` is not been updated to 2021-12-26 and is now out of sync with the date in the other files.

Please fix on commit, rest is OK

**Pull request merged**